### DISCUSSION TITLE
Is there a way to edit a new version the map using Maputnik instead?

### DISCUSSION THREAD

**Discussion by Victorhg7** [osm-americana/openstreetmap-americana#1151] [Q&A] [answered]
I am thinking on using the OSM-Americana map as a starter for a custom map style, especially because how it shows Freeways and Tollways with their own color (rather than both having the same color) but currently Maputnik forces me to color Freeways and Tollways the same color when starting a style from their site. If anyone knows how to load the OSM-Americana map style into Maputnik please let me know.

**Comment by ZeLonewolf** [comment, marked as answer]
The style JSON file can be loaded from here:
https://americanamap.org/style.json

**Reply by mrmax3d** [reply]
Can you also explain to a newbie how to get the highway sheilds to work using this style?

**Reply by 1ec5** [reply]
You’ll need to install [a separate shield library](https://www.npmjs.com/package/@americana/maplibre-shield-generator) and configure it using [these instructions](https://github.com/osm-americana/openstreetmap-americana/tree/c4829f2c91099c3b5619fff829a390133ea08a53/shieldlib#shield-rendering-workflow).

**Reply by mrmax3d** [reply]
Thanks Minh for your response. So I am quite new at this and need to produce a dozen or so maps in a short amount of time. I will bespending some time to learn how to customize maps but, for now, do you know of one similar to Americana that has the highway shields installed and configured? I'm open to a variety of styles, it's important that i have the sheilds.

**Reply by 1ec5** [reply]
Is there a particular region or highway system that you’re interested in? If you don’t need the repertoire of shields to be as comprehensive as what OSM Americana has, there are [several other implementations](https://wiki.openstreetmap.org/wiki/United_States/Road_signs#Implementations) of American route shields and many more implementations of more generic “shields” suitable for, say, Europe. If you’re comfortable with a stylesheet editor such as [Maputnik](https://maputnik.github.io/), you could use OSM Americana as a starting point and add your own custom shield artwork using [OpenMapTiles route information](https://openmaptiles.org/schema/#route_1_network). OSM Americana defaults to the OSMUS tile service, which is one of several providers of OpenMapTiles-compatible tiles.
